chess-AI/options.py

```diff
@@ -14,20 +14,19 @@ class TrainOptions:
         device: the device being used to train (either CPU or GPU)
         save_path: path for model checkpointing
         num_saved_models: number of models to store
+        m_saving: parameters for model saving
     """
 
     def __init__(self, learning_rate, momentum, weight_decay, epochs, batch_size, games, device,
-                 save_path, num_saved_models, overwrite):
+                 m_saving):
```

Comment:
Please add a comment describing `m_saving`

Reply:
In the attributes there is a comment for m_saving, we can have more detailed explanation of m_saving in our documentation. It is alot to put in a comment to describe the whole dictionary.

Reply:
We decided to make a class for model saving and dataset saving